tools/fork-network/src/cli.rs

```diff
@@ -527,6 +532,52 @@ impl ForkNetworkCommand {
         return Ok(());
     }
 
+    /// Creates epoch config overrides since `first_version` and places them
+    /// in `home_dir`.
+    fn override_epoch_configs(
+        &self,
+        first_version: ProtocolVersion,
+        num_seats: &Option<NumSeats>,
+        home_dir: &Path,
+    ) -> anyhow::Result<EpochConfig> {
+        let epoch_config_dir = home_dir.join("epoch_configs");
+        if epoch_config_dir.exists() {
+            std::fs::remove_dir_all(epoch_config_dir.clone())?;
+        }
+        std::fs::create_dir_all(epoch_config_dir.clone()).with_context(|| {
+            anyhow::anyhow!("Failed to create directory {:?}", epoch_config_dir)
+        })?;
+
+        let base_epoch_config_store =
+            EpochConfigStore::for_chain_id(near_primitives::chains::MAINNET, None)
```

Comment:
Could be good to be aware somehow of what chain we forked to create the state in the first place (could have been testnet, or some other random net). Maybe could just derive this epoch config from the starting genesis? Anyway it's not a big deal for now, so we can just worry about this later. But maybe add a TODO?

Reply:
1) Do we ever need to fork any chain other than mainnet?
2) Creating epoch config from genesis is actually what we want to get rid of: #11265.

Reply:
1. eh I guess it's not impossible. maybe one day something interesting happens on testnet and we want to try it out in forknet
2. +1